contrib/fixup-libgfortran.sh

```diff
@@ -41,6 +42,20 @@ find_shlib()
     fi
 }
 
+find_shlib_dir()
+{
+    # Usually, on platforms like OSX we get full paths when linking.  However,
+    # if we are inspecting, say, BinaryBuilder-built OpenBLAS libraries, we will
+    # only get something like `@rpath/libgfortran.5.dylib` when inspecting the
+    # libraries.  We can, as a last resort, ask `$FC` directly what the full
+    # filepath for this library is, but only if we don't have a direct path to it:
+    if [ $(dirname "$1") == "@rpath" ]; then
```

Comment:
Isn't `==` also a Bash thing? And what about `${FC:-gfortran}`?

Reply:
`${FC:-gfortran}` should be fine, [according to this](http://pubs.opengroup.org/onlinepubs/009696899/utilities/xcu_chap02.html).